From bf4f8919df8ee88e356b407bb84ed818ebfb407b Mon Sep 17 00:00:00 2001
From: Howard Huang <howardhuang@meta.com>
Date: Wed, 22 Jan 2025 10:12:15 -0800
Subject: [PATCH] Fix test_modules_can_be_imported (#145387)

`test_modules_can_be_imported` test is currently failing due to a few missing private modules and this PR gets it working before I start to clean up the public allow list
Pull Request resolved: https://github.com/pytorch/pytorch/pull/145387
Approved by: https://github.com/albanD
---
 test/test_public_bindings.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/test_public_bindings.py b/test/test_public_bindings.py
index c4e819704f17b..61065fcd629d2 100644
--- a/test/test_public_bindings.py
+++ b/test/test_public_bindings.py
@@ -308,6 +308,7 @@ def onerror(modname):
             "torch.onnx._internal.exporter._reporting",
             "torch.onnx._internal.exporter._schemas",
             "torch.onnx._internal.exporter._tensors",
+            "torch.onnx._internal.exporter._torchlib.ops",
             "torch.onnx._internal.exporter._verification",
             "torch.onnx._internal.fx._pass",
             "torch.onnx._internal.fx.analysis",
@@ -377,6 +378,7 @@ def onerror(modname):
             "torch.distributed._spmd.experimental_ops",
             "torch.distributed._spmd.parallel_mode",
             "torch.distributed._tensor",
+            "torch.distributed._tools.sac_ilp",
             "torch.distributed.algorithms._checkpoint.checkpoint_wrapper",
             "torch.distributed.algorithms._optimizer_overlap",
             "torch.distributed.rpc._testing.faulty_agent_backend_registry",
